src/test/java/gov/acwi/wqp/etl/EtlBiodataIT.java

```diff
@@ -0,0 +1,39 @@
+package gov.acwi.wqp.etl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.JobExecution;
+
+import com.github.springtestdbunit.annotation.ExpectedDatabase;
+import com.github.springtestdbunit.assertion.DatabaseAssertionMode;
+
+public class EtlBiodataIT extends BiodataBaseFlowIT {
+
+	@Test
```

Comment:
The database setup and expected should be the same as for the monitoringLocationFlowTest below.
It will expand as more objects are added and should eventually include all the code and summary tables similar to the (expanding) EtlStewardsIt.

Reply:
Roger that, updated (and will continue to update it).